### PR TITLE
Dataexfil

### PR DESCRIPTION
**Pull request by chandrasekarsrinivasan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServiceEndpointPolicy": {
+      "properties": {
```
Comment (chandrasekarsrinivasan) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: Add name

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServiceEndpointPolicy"
+          },
+          "description": "A list of service endpoint policies that exists in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: Please mark `nextLink` properties as readonly

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for ListServiceEndpointPolicies API service call."
+    },
+    "serviceEndpointPolicyDefinitionsFormat": {
+      "name": {
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: Please check if these properties are readonly

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+          "description": "service endpoint name."
+        },
+        "serviceResources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of service resources."
+        },
+        "provisioningState": {
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: I understand this is usually an enum. Must be readonly.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments.
Please address linter/model validator violations if any

**Review by dsgouda — CHANGES_REQUESTED** [review]
Added additional comments based on the example

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting ServiceEndpointPolicy resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicy"
+            }
+          }
+        },
+        "x-ms-examples": {
```
Comment (chandrasekarsrinivasan) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: example required?

**Comment by dsgouda** [comment]
Closing this. @MikhailTryakhov FYI

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json (hunk 2 above)
Comment (chandrasekarsrinivasan) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: done.

File: specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json (hunk 4 above)
Comment (chandrasekarsrinivasan) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json]: done

**Pull request closed without merging**